externals/kyuubi-jdbc-engine/src/test/scala/org/apache/kyuubi/engine/jdbc/mysql/WithMySQLContainer.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kyuubi.engine.jdbc.mysql
+
+import com.dimafeng.testcontainers.{MySQLContainer, SingleContainer}
+import org.testcontainers.utility.DockerImageName
+
+import org.apache.kyuubi.engine.jdbc.WithJdbcServerContainer
+
+trait WithMySQLContainer extends WithJdbcServerContainer {
```

Comment:
You dont have to wrap the Mysql container again.
For creating/starting and stopping containers,  the trait`TestContainerForAll` is perfect for the test suites.
For setting and getting query string, use the `testQueryString` of `MySQLContainer`.
For image name and other common settings, they are all available in the `MySQLContainer` object.